cmd/mouseRecon/testdata/allPossibleVisualize0point8.txt

```diff
@@ -0,0 +1,25 @@
+Start: 0. refCounter: 0. alignCounter: 0
```

Comment:
This file doesnt seem to be used in the test file, but maybe it is here because you used it to derive another file that you are testing?